src/matrixfree_operators.jl

```diff
@@ -6,7 +6,7 @@ mutable struct MatrixFreeOperator{F,N,S,O} <: AbstractMatrixFreeOperator{F}
   opnorm::O
   ishermitian::Bool
   function MatrixFreeOperator(f::F, args::N;
-                              size=nothing, opnorm=nothing, ishermitian=false) where {F,N}
+                              size=nothing, opnorm=true, ishermitian=false) where {F,N}
```

Comment:
why is this true/false?

Reply:
`opnorm` is defaulted to be one, as `opnorm` is only used to test relative tolerance in `expv`.